openedx/core/djangoapps/content_libraries/api/libraries.py

```diff
@@ -712,3 +722,42 @@ def get_backup_task_status(
         result['file'] = artifact.file
 
     return result
+
+
+def _transform_lib_permission_to_authz_permission(permission: str) -> str:
+    """
+    Transform a content library permission to an openedx-authz permission.
+    """
+    mapping = {
+        permissions.CAN_CREATE_CONTENT_LIBRARY: 'create_library',
+        permissions.CAN_DELETE_THIS_CONTENT_LIBRARY: 'delete_library',
+        permissions.CAN_EDIT_THIS_CONTENT_LIBRARY: 'edit_library_content',
+        permissions.CAN_EDIT_THIS_CONTENT_LIBRARY_TEAM: 'manage_library_team',
+        permissions.CAN_VIEW_THIS_CONTENT_LIBRARY: 'view_library',
+        permissions.CAN_VIEW_THIS_CONTENT_LIBRARY_TEAM: 'view_library_team',
+    }
+    return mapping.get(permission, permission)
+
+
+def require_authz_lib_permission(
```

Comment:
This function is a bit unclear, so two suggestions:
1. Could we rename it? The current name doesn’t give enough context about what it does.
2. The docstring could also use some improvement to better explain its purpose.